docker-bake.hcl

```diff
@@ -0,0 +1,72 @@
+// Go version
+variable "GO_VERSION" {
+  default = "1.17"
+}
```

Comment:
Is it possible to "skip" this variable if it's not set, so that the default from the Dockerfile can be used?

Reply:
We can make the default empty sure.